modules/structs/issue.go

```diff
@@ -28,6 +28,7 @@ const (
 type PullRequestMeta struct {
 	HasMerged bool       `json:"merged"`
 	Merged    *time.Time `json:"merged_at"`
+	IsDraft   bool       `json:"draft"`
```

Comment:
I think `IsWIP` is a better name.

Reply:
Renamed `IsDraft` to `IsWorkInProgress` for consistency with [PullRequest.IsWorkInProgress](https://github.com/go-gitea/gitea/blob/2a50d780f93957223da9494c61bb3181ffa7ff65/models/issues/pull.go#L689)

Renamed json field `draft` to `wip` for consistency with the terminology used in the docs.